airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/fix/commands.py

```diff
@@ -90,10 +90,10 @@ async def all_fix(ctx: click.Context):
 @pass_pipeline_context
 @click_ignore_unused_kwargs
 async def java(ctx: ClickPipelineContext) -> CommandResult:
-    """Format java, groovy, and sql code via spotless."""
+    """Format java code via spotless in maven."""
     dagger_client = await ctx.get_dagger_client(pipeline_name="Format java")
     container = format_java_container(dagger_client)
-    format_commands = ["./gradlew spotlessApply --scan"]
+    format_commands = ["mvn -f spotless-maven-pom.xml spotless:apply"]
```

Comment:
if we do it via maven directly and not gradle, do we still have the same definition of spotless as the one in our build.gradle file? 

Specifically talking about the extra steps we add to java in addition to the `target` here:
```
    // Apply spotless formatting.
    apply plugin: 'com.diffplug.spotless'
    spotless {
        def javaTarget = createFormatTarget('**/*.java')
        if (!javaTarget.isEmpty()) {
            java {
                target javaTarget
                importOrder()
                eclipse('4.21').configFile(rootProject.file('tools/gradle/codestyle/java-google-style.xml'))
                removeUnusedImports()
                trimTrailingWhitespace()
            }
        }
        def groovyGradleTarget = createFormatTarget('**/*.gradle')
        if (!groovyGradleTarget.isEmpty()) {
            groovyGradle {
                target groovyGradleTarget
            }
        }
    }
```

Reply:
I see the codestyle file [here](https://github.com/airbytehq/airbyte/pull/32999/files#diff-5736b36ac856cdc452b2d1ac119ad9381834adf08cc6d9acd04b6d46e426bd33R31) but not the other methods. Looking at it maybe they are custom and not spotless at all?

Reply:
>  do we still have the same definition of spotless as the one in our build.gradle file

Good catch! I think I can add these to the pom.

Reply:
Done. I checked and spotless's feature coverage for gradle and maven are almost identical.

Reply:
There are still references to spotlessCheck tasks inside the gradle build so I'm not going to remove it just yet but I'm looking forward to it!